src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/TimeOutLicencesService.kt

```diff
@@ -76,6 +80,22 @@ class TimeOutLicencesService(
           eventDescription = "${reason ?: "Licence automatically timed out"} for ${licence.forename} ${licence.surname}",
         ),
       )
+
+      // previously approved now edited licence that is timed out
+      if (hardStopEnabled && licence.versionOfId != null) {
```

Comment:
presuambly we don't need to add this flag as it we would only run this job if we were enabling hardstop

Reply:
if we were to use it - would make more sense exiting early at the start of the job